### DISCUSSION TITLE
Using org-remark with elfeed

### DISCUSSION THREAD

**Discussion by jackmac92** [nobiot/org-remark#37] [Q&A]
Hey I'd like to be able to take notes on an article I read via elfeed

I know elfeed isn't using org mode to display, but since you can take notes on any text file I'm hopeful

But elfeed doesn't show the article in a file buffer

The most "actionable" high level question for me is, what would be necessary to leave comments on non-file buffers?

**Comment by nobiot** [comment]
You might like to refer to what @vedang has done with his [PR](https://github.com/nobiot/org-remark/pull/27). He has enabled org-remark for website via EWW, which does not visit a file in your system.

I don't use EWW or Elfeed so would need to rely on your knowledge and skills.

**Reply by jackmac92** [reply]
This is exactly what I was looking for thanks! 

(sorry for the mark as answer spam, decided to unmark in case others are interested in the original question)

**Reply by nobiot** [reply]
Does this approach work for you for elfeed? I’m wondering if highlights’ locations correctly persist between Emacs sessions, and when elfeed updates the feed.
